sharepoint/sharepoint-ps/sharepoint-online/Get-SPOMigrationJobProgress.md

```diff
@@ -138,7 +138,7 @@ Accept wildcard characters: False
 
 ### -EncryptionParameters
 
-PARAMVALUE: EncryptionParameters
+An EncryptionParameters object. See [New-SPOMigrationEncryptionParameters](https://docs.microsoft.com/powershell/module/sharepoint-online/new-spomigrationencryptionparameters) for more information.
```

Comment:
I *think* you may need to put the plain URL rather than markdown, i.e. "See New-SPOMigration[...] https://docs[...]."

Reply:
For any special reason ? Thanks for your feedback.

Reply:
Hm I suppose in PoSh it may just print out the literal markdown...